static/js/babelhook.js

```diff
@@ -7,7 +7,7 @@ require('jsdom-global')(undefined, {
 });
 
 // We need to explicitly change the URL when window.location is used
-const changeURL = require('jsdom').changeURL;
+const changeURL = require('jsdom/lib/old-api').changeURL;
```

Comment:
It doesn't fully support changing `window.location` in a straightforward way, unfortunately